app/Helpers/SearchHelper.php

```diff
@@ -18,7 +18,7 @@ class SearchHelper
     public static function searchContacts($query, $limitPerPage, $order)
     {
         $needle = $query;
-        $accountId = Auth::user()->account_id;
+        $accountId = request()->get('accountId') ? request()->get('accountId') : Auth::user()->account_id;
```

Comment:
This is not a good idea, it can open a security breach.

Reply:
It will check records only for login user based on login user account, we need to duplicate code for test case I guess.